modules/is_master/manifests/params.pp

```diff
@@ -0,0 +1,71 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#----------------------------------------------------------------------------
+
+class is_master::params {
+
+  $user = 'wso2carbon'
+  $user_group = 'wso2'
+  $product = 'wso2is'
+  $product_version = '5.7.0'
+  $service_name = 'wso2is'
+
+  # Define the templates
+  $start_script_template = 'bin/wso2server.sh'
+  $build_script_template = 'build.sh'
```

Comment:
IMO, it is good to have a brief explanation on the build script here as a comment. WDYT?

Reply:
+1 will add description

Reply:
Resolved https://github.com/wso2/puppet-is/pull/58/commits/b8ba5cf3de87228146971ff913f98bd16021353b